.devcontainer/README.md

```diff
@@ -0,0 +1,100 @@
+# Dev Container Setup for ComfyStream
+
+This guide will help you set up and run a development container for ComfyStream using Visual Studio Code (VS Code).
+
+## Prerequisites
+
+- [Docker](https://www.docker.com/get-started)
+- [Visual Studio Code](https://code.visualstudio.com/)
+- [VS Code Remote - Containers extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers)
+
+## Building the Base Container
+
+To build the base container, run the following command in your terminal:
+
+```sh
+docker build -f docker/Dockerfile.base -t livepeer/comfyui-base:latest .
+```
+
+## Using the Pre-built Base Container
+
+Most users will configure host paths for models in `devcontainer.json` and use the pre-built base container. Follow these steps:
+
+1. Pull the pre-built base container:
+
+```sh
+docker pull livepeer/comfyui-base:latest
+```
+
+2. Configure the host paths for models in the `devcontainer.json` file.
+
+3. Re-open the workspace in the dev container using VS Code:
+    - Open the Command Palette (`Ctrl+Shift+P` or `Cmd+Shift+P` on macOS).
+    - Select `Remote-Containers: Reopen in Container`.
+
+## Configuration
+
+Create a directory to store the models and engines
+```sh
+sudo mkdir -p /models/ComfyUI--models && sudo mkdir -p /models/ComfyUI--output
+sudo chown -R $USER /models/ComfyUI--models && sudo chown -R $USER /models/ComfyUI--output
+```
+
+If you have a different path for models, ensure your `devcontainer.json` is properly configured to map the correct host path to your models. Here is an example configuration:
+
+```json
+{
+    "name": "ComfyStream Dev Container",
+    "image": "livepeer/comfyui-base",
+    "mounts": [
+        "source=/path/to/your/models,target=/ComfyUI/models,type=bind"
+    ],
+    "workspaceFolder": "/workspace"
```

Comment:
@eliteprox this could confuse developers since you also use `--workspace` to specify the ComfyUI workspace. Maybe we can use `${workspaceFolder}` instead?

Reply:
Removed the duplicate runArg `--workspace` as this `workspaceFolder` mount is the correct way to do it